docs/content/preview/explore/observability/active-session-history.md

```diff
@@ -91,47 +91,64 @@ List of wait events by the wait event components.
+| Wait Event Class | Wait Event | Wait Event Type | Wait Event Aux | Description |
 |:---------------- | :--------- |:--------------- | :--- | :---------- |
-| Common | OnCpu_Passive | CPU | | Waiting for a thread to pick it up |
-| Common | OnCpu_Active | CPU |  | RPC is being actively processed on a thread |
-| Common | ResponseQueued | Network | | Waiting for response to be transferred |
-| Tablet | AcquiringLocks | Lock | \<tablet&#8209;id>| Taking row-wise locks. May need to wait for other rpcs to release the lock. |
-| Tablet | MVCC_WaitForSafeTime | Lock | \<tablet-id>| Waiting for the SafeTime to be at least the desired read-time. |
-| Tablet | BackfillIndex_WaitForAFreeSlot | Lock | \<tablet-id> | Waiting for a slot to open if there are too many backfill requests at the same time. |
-| Tablet | CreatingNewTablet | I/O  | \<tablet-id>| Creating a new tablet may involve writing metadata files, causing I/O wait.  |
-| Tablet | WaitOnConflictingTxn | Lock | \<tablet-id>| Waiting for the conflicting transactions to complete. |
-| Consensus | WAL_Append | I/O | \<tablet-id>| Persisting Wal edits |
-| Consensus | WAL_Sync | I/O | \<tablet-id>| Persisting Wal edits |
-| Consensus | Raft_WaitingForReplication | Network | \<tablet-id>| Waiting for Raft replication |
-| Consensus | Raft_ApplyingEdits | Lock/CPU | \<tablet-id>| Applying the edits locally |
-| RocksDB  | BlockCacheReadFromDisk | I/O  | \<tablet-id>| Populating block cache from disk |
-| RocksDB | Flush  | I/O | \<tablet-id> | Doing RocksDB flush  |
-| RocksDB | Compaction | I/O | \<tablet-id>| Doing RocksDB compaction |
-| RocksDB | RateLimiter | I/O | | Slow down due to rate limiter throttling access to disk |
+| Common | OnCpu_Passive | CPU | | An RPC or task is waiting for a thread to pick it up. |
+| Common | OnCpu_Active | CPU |  | An RPC or task is being actively processed on a thread. |
+| Common | Idle | WaitOnCondition | | The raft log appender/sync thread is idle. |
+| Common | Rpc_Done | WaitOnCondition | | An RPC is done and waiting for the reactor to send the response to a YSQL/YCQL backend. |
+| Common | RetryableRequests_SaveToDisk | DiskIO | | The in-memory state of the retryable requests is being saved to the disk. |
+| TabletWait | MVCC_WaitForSafeTime | WaitOnCondition | | A read/write RPC is waiting for the safe time to be at least the desired read-time. |
+| TabletWait | LockedBatchEntry_Lock | WaitOnCondition | | A read/write RPC is waiting for a DocDB row-level lock. |
+| TabletWait | BackfillIndex_WaitForAFreeSlot | WaitOnCondition | | A backfill index RPC is waiting for a slot to open if there are too many backfill requests at the same time. |
+| TabletWait | CreatingNewTablet | DiskIO | | The CreateTablet RPC is creating a new tablet, this may involve writing metadata files, causing I/O wait. |
+| TabletWait | SaveRaftGroupMetadataToDisk | DiskIO | | The RAFT/tablet metadata is being written to disk, generally during snapshot or restore operations. |
```

Comment:
Pasting it here since the other conversation seems to be hidden

> In other places in documentation, the term is written as "Raft", why was this changed to "RAFT" ?

Reply:
agreed. Reverting the change.